README.md

```diff
@@ -251,26 +253,38 @@ Note that the final import of `REISE` may take a couple of minutes to complete.
 ## Usage (Julia)
 Installation registers a package named `REISE`. Following Julia naming conventions, the `.jl` is dropped. The package can be imported using: `import REISE` to call `REISE.run_scenario()`, or `using REISE` to call `run_scenario()`.
 
-To run a scenario which starts at the `1`st hour of the year, runs in `3` intervals of `24` hours each, loading input data from your present working directory (`pwd()`) and depositing results in the folder `output`, call:
-```julia
-REISE.run_scenario(;
-    interval=24, n_interval=3, start_index=1, outputfolder="output",
-    inputfolder=pwd())
-```
-An optional keyword argument `num_segments` controls the linearization of cost curves into piecewise-linear segments (default is 1). For example:
+Running a scenario requires the following inputs:
+- `interval`: the length of each simulation interval (hours).
+- `n_interval`: the number of simulation intervals.
+- `start_index`: the hour to start the simulation, representing the row of the time-series profiles in `demand.csv`, `hydro.csv`, `solar.csv`, and `wind.csv`.
+Note that unlike some other programming languages, Julia is 1-indexed, so the first index is `1`.
+- `inputfolder`: the directory from which to load input files.
+- `optimizer_factory`: an argument which can be passed to `JuMP.Model` to create a new model instance with an attached solver.
+Be sure to pass the factory itself (e.g. `GLPK.Optimizer`) rather than an instance (e.g. `GLPK.Optimizer()`). See the [JuMP.Model documentation] for more information.
+
+As an example, to run a scenario which starts at the `1`st hour of the year, runs in `3` intervals of `24` hours each, loading input data from your present working directory (`pwd()`), using the `GLPK` solver, call:
 ```julia
 REISE.run_scenario(;
-    interval=24, n_interval=3, start_index=1, outputfolder="output",
-    inputfolder=pwd(), num_segments=3)
+    interval=24, n_interval=3, start_index=1, inputfolder=pwd(), optimizer_factory=GLPK.Optimizer
```

Comment:
Does this section need an `import GLPK`?

Reply:
Done.